kubee2etests/helpers_and_globals.py

```diff
@@ -110,3 +112,52 @@ def load_kubernetes():
             logging.error("Not able to use in-cluster config: %s", err)
             sys.exit(1)
     return incluster
+
+def add_error(self, err):
+    error_list = [error[0] for error in self.errors]
```

Comment:
ick, didn't fully realise how much these were intertwined into self. I guess pass in `err, errors` here then return errors when you're done updating it rather than depending on self.

Or we could pull these into a class both ApiMixin and RunnerBase inherit from which might be less messy.